src/cpu/minor/BaseMinorCPU.py

```diff
@@ -249,31 +269,7 @@ class MinorDefaultMiscFU(MinorFU):
     opLat = 1
 
 
-class MinorDefaultVecFU(MinorFU):
-    opClasses = minorMakeOpClassSet(
-        [
-            "VectorUnitStrideLoad",
-            "VectorUnitStrideStore",
-            "VectorUnitStrideMaskLoad",
-            "VectorUnitStrideMaskStore",
-            "VectorStridedLoad",
-            "VectorStridedStore",
-            "VectorIndexedLoad",
-            "VectorIndexedStore",
-            "VectorUnitStrideFaultOnlyFirstLoad",
-            "VectorWholeRegisterLoad",
-            "VectorWholeRegisterStore",
-            "VectorIntegerArith",
-            "VectorFloatArith",
-            "VectorFloatConvert",
-            "VectorIntegerReduce",
-            "VectorFloatReduce",
-            "VectorMisc",
-            "VectorIntegerExtension",
-            "VectorConfig",
-        ]
-    )
-    opLat = 1
+MinorDefaultVecFU = MinorDefaultFloatSimdFU
```

Comment:
I'd remove it completely, by default we make Float and Simd opClasses go to the same FU. It also needs to be removed from the `MinorDefaultFUPool`.